developerportal/apps/common/migrations/0001_drop_social_auth_app_tables.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.6 on 2019-10-24 12:27
+"""
+Drop django-social-auth tables from the database, if they still exist,
+following the removal of it from the codebase.
+"""
+
+import logging
+
+from django.db import connection
+from django.db import migrations
+
+logger = logging.getLogger(__name__)
+
+
+def drop_social_auth_tables(apps, schema_migration):
+    # We're expecting them to be empty, with no cascading side-effects
+    table_names = [
+        "social_auth_usersocialauth",
+        "social_auth_partial",
+        "social_auth_association",
+        "social_auth_code",
+        "social_auth_nonce",
+    ]
+    logger.info("Dropping redundant DB tables %s", table_names)
+    with connection.cursor() as cursor:
+        for table_name in table_names:
+            query = f"DROP TABLE IF EXISTS {table_name}"
+            logger.info(f"Query: {query}")
+            cursor.execute(query)
+
```

Comment:
This works when the tables exist and also when they have already gone (or never existed in the first place - eg, in CI).

An example output from tests running:
```
Creating test database for alias 'default'...
INFO:developerportal.apps.common.migrations.0001_drop_social_auth_app_tables:Dropping redundant DB tables ['social_auth_usersocialauth', 'social_auth_partial', 'social_auth_association', 'social_auth_code', 'social_auth_nonce']
INFO:developerportal.apps.common.migrations.0001_drop_social_auth_app_tables:Query: DROP TABLE IF EXISTS social_auth_usersocialauth
INFO:developerportal.apps.common.migrations.0001_drop_social_auth_app_tables:Query: DROP TABLE IF EXISTS social_auth_partial
INFO:developerportal.apps.common.migrations.0001_drop_social_auth_app_tables:Query: DROP TABLE IF EXISTS social_auth_association
INFO:developerportal.apps.common.migrations.0001_drop_social_auth_app_tables:Query: DROP TABLE IF EXISTS social_auth_code
INFO:developerportal.apps.common.migrations.0001_drop_social_auth_app_tables:Query: DROP TABLE IF EXISTS social_auth_nonce
System check identified no issues (0 silenced).
```

Reply:
I've ticketed-up removing this migration once it's done its duty: https://github.com/mdn/developer-portal/issues/475